packages/flutter_plugin_android_lifecycle/CHANGELOG.md

```diff
@@ -1,15 +1,8 @@
-## 2.0.0-nullsafety.2
+## 2.0.0
```

Comment:
it's funny to see this package bumping major versions, since that was meant to indicate a breaking change to the Android embedding. 🤔